source/derived.tex

```diff
@@ -147,16 +147,13 @@
 a derived class and its base class subobjects can be represented by a
 directed acyclic graph (DAG) where an arrow means ``directly derived
 from''. A DAG of subobjects is often referred to as a ``subobject
-lattice''.
+lattice''. The arrows need not have a physical representation in memory.
```

Comment:
Oh, sorry, I re-pushed before addressing this second review comment. I'll address this one too.